src/Seq.Input.RabbitMQ/RabbitMQInput.cs

```diff
@@ -19,6 +19,12 @@ public class RabbitMQInput : SeqApp, IPublishJson, IDisposable
             HelpText = "The hostname on which RabbitMQ is running. The default is `localhost`.")]
         public string RabbitMQHost { get; set; } = "localhost";
 
+        [SeqAppSetting(
+            DisplayName = "RabbitMQ vhost",
```

Comment:
To match the RabbitMQ docs this should probably be `"RabbitMQ Virtual Host"`.

Reply:
Changed